internal/controllers/machineset/machineset_controller.go

```diff
@@ -680,7 +680,6 @@ func (r *Reconciler) syncReplicas(ctx context.Context, s *scope) (ctrl.Result, e
 	switch {
 	case diff < 0:
 		diff *= -1
-		log.Info(fmt.Sprintf("MachineSet is scaling up to %d replicas by creating %d machines", *(ms.Spec.Replicas), diff), "replicas", *(ms.Spec.Replicas), "machineCount", len(machines))
```

Comment:
Do we want to remove that log line or just set some verbosity level instead if its too verbose? Or is it obsolete?